webapp/src/utils.ts

```diff
@@ -287,17 +287,11 @@ export function hasExperimentalFlag() {
     return window.localStorage.getItem('calls_experimental_features') === 'on';
 }
 
-export function getUsersList(profiles: UserProfile[]) {
-    if (profiles.length === 0) {
-        return '';
-    }
-    if (profiles.length === 1) {
-        return getUserDisplayName(profiles[0]);
+export function split<T>(list: T[], i: number, pad = false): [list: T[], overflowed?: T[]] {
+    if (list.length <= i + (pad ? 1 : 0)) {
+        return [list];
```

Comment:
Could you explain the purpose of the pad? (I must be slow today, heh.)

Reply:
Not the greatest name for that arg in all honesty, not sure what to name it exactly, but it means "pad the threshold by 1"

It allows for:
`a, b, c` (+1 to) `a, b, & 2 others`
instead of: 
`a, b, c` (+1 to) `a, b, c, & 1 other`

Reply:
- Would it be possible to use the literal `and` like in the other lists? Not exactly sure why it was designed that way but it seems inconsistent.
-  Also I'd probably remove the comma before that if possible because it reads a little awkward.

Reply:
1. Done
2. The oxford comma is mostly out of our control and depends on the rules of the locale. `de` doesn't use one in this api, whereas `en-US` does use an oxford comma for this api, and `en-GB` on the other hand doesn't. While it's technically possible to remove it via `formatToParts` and reconstructing it, I would vote to let `formatList` do its thing.

In the future it'd be nice to be able to honor more granular locales better in MM for secondary formatting preferences like this—as well as date/time, units, etc.

Reply:
Thanks! Not a big deal just personally found it odd but it's borderline OCD stuff.